src/Car.js

```diff
@@ -0,0 +1,60 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+import { ERROR_MESSAGE } from './constant.js';
+import { errorString } from './util.js';
+
+class Car {
+  #name;
+  #position;
+
+  static MAX_CAR_NAME_LENGTH = 5;
+  static PRINT_POSITION_FORM = '-';
+
+  constructor(name) {
+    this.#validateName(name);
+
+    this.#name = name;
+    this.#position = 0;
+  }
+
+  #validateName(name) {
+    if (name.length === 0) {
+      throw new Error(errorString(ERROR_MESSAGE.MIN_CAR_NAME_LENGTH));
+    }
+
+    if (name.length > Car.MAX_CAR_NAME_LENGTH) {
+      throw new Error(errorString(ERROR_MESSAGE.MAX_CAR_NAME_LENGTH));
+    }
+  }
+
+  move() {
+    const canMove = this.#isMove();
+
+    if (canMove) {
+      this.#position += 1;
+    }
+  }
+
+  #isMove() {
+    return Random.pickNumberInRange(0, 9) >= 4;
+  }
```

Comment:
```suggestion
  #isMove(minRandomNumber, maxRandomNumber, moveCondition) {
    return Random.pickNumberInRange(minRandomNumber, maxRandomNumber) >= moveCondition;
  }
```
나중에 게임의 요구사항이 변했을 때 유연하게 사용할 수 있게 숫자들을 인자로 받는 것도 좋은 것 같아요!